documentation/docs/20-core-concepts/30-form-actions.md

```diff
@@ -140,7 +140,7 @@ export const actions = {
 
 #### Validation errors
 
-If the request couldn't be processed because of invalid data, you can return validation errors — along with the previously submitted form values — back to the user so that they can try again. The `invalid` function lets you return an HTTP status code (typically 400 or 422, in the case of validation errors) along with the data:
+If the request couldn't be processed because of invalid data, you can return validation errors — along with the previously submitted form values — back to the user so that they can try again. The `invalid` function lets you return an HTTP status code in `$page.status` (typically 400 or 422, in the case of validation errors) along with the data in `form`:
```

Comment:
```suggestion
If the request couldn't be processed because of invalid data, you can return validation errors — along with the previously submitted form values — back to the user so that they can try again. The `invalid` function lets you return an HTTP status code (typically 400 or 422, in the case of validation errors) along with the data. The status code is available through `$page.status`, the data through `form`:
```

Reply:
Wow you're quick; I've applied this change, but added 'and' to the final list item since I feel it reads bit more naturally.